$OpenBSD: patch-src_hamster-applet_py,v 1.1 2010/04/24 11:08:21 ajacoutot Exp $

- Fix path to python
- The bindtextdomain function isn't present in our python.

--- src/hamster-applet.py.orig	Mon Mar 29 13:45:01 2010
+++ src/hamster-applet.py	Fri Apr 23 12:09:25 2010
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!${MODPY_BIN}
 # - coding: utf-8 -
 
 # Copyright (C) 2007, 2008 Toms BauÄ£is <toms.baugis at gmail.com>
@@ -96,12 +96,6 @@ if __name__ == "__main__":
 
         # Setup i18n
         locale_dir = os.path.abspath(os.path.join(defs.DATA_DIR, "locale"))
-        for module in (gettext, locale):
-            module.bindtextdomain('hamster-applet', locale_dir)
-            module.textdomain('hamster-applet')
-
-            if hasattr(module, 'bind_textdomain_codeset'):
-                module.bind_textdomain_codeset('hamster-applet','UTF-8')
 
         gtk.window_set_default_icon_name("hamster-applet")
 
